### DISCUSSION TITLE
Add the ability to set a display name

### DISCUSSION THREAD

**Discussion by computertechnikberger** [rustdesk/rustdesk-server-pro#426] [Ideas]
Maybe I'm missing it, but it would be nice to have the option to set a display name. We use RustDesk to access customer PCs. Unfortunately, only the user name is currently displayed. It would be clearer for our customers if, for example, our first and last name and the company were displayed. We used AnyDesk before. It is possible there too. What do you think?

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/client-configuration/advanced-settings/#display-name

**Reply by PC-Support-Services** [reply]
Does display-name support capital letters and strings?

ie 

display-name=Acme Co Ltd _or_
display-name="Acme Co Ltd"

?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/issues/875 we will add a "display name" field to user.
